opta/commands/config.py

```diff
@@ -0,0 +1,62 @@
+import click
+from click_didyoumean import DYMGroup
+
+from opta.core.aws import AWS
+from opta.core.azure import Azure
+from opta.core.cloud_client import CloudClient
+from opta.core.gcp import GCP
+from opta.exceptions import UserErrors
+from opta.utils import logger
+
+
+@click.group(cls=DYMGroup)
+def config() -> None:
+    """
+    (Beta) Manage opta configurations.
+    """
+    pass
+
+
+cloud_option = click.option(
+    "--cloud",
+    help="The cloud provider to use",
+    required=True,
+    type=click.Choice(["aws", "google", "azurerm"]),
+)
+
+
+@config.command()
+@cloud_option
+def view(cloud: str) -> None:
+    """
+    View the opta configuration file for a given cloud provider.
+    Use the current cloud credentials to fetch the remote storage.
+
+    Examples:
+
+        - View opta configurations:
```

Comment:
we don't need this line, create extra ident and the command itself is self descriptive
check
`opta secret view -h `